scripts/github_actions/README.md

```diff
@@ -0,0 +1 @@
+These scripts are intended to be run on github-actions CI and may not run correctly outside of that environment.
```

Comment:
Should these move into the .github folder instead?

Reply:
i have no idea what best practices for the .github folder are... I found one article on the internet that didn't explicitly list sketchy python scripts that could be reimplemented as a github action explicitly but you opted not to for verisimilitudes sake in the list of things but maybe their list wasn't exactly comprehensive.